Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -561,6 +644,216 @@ def use_homebrew_curl?(formula_or_cask, url)
       homebrew_curl_root_domains.include?(url_root_domain)
     end
 
+    # Identifies the latest version of the resource in a given Formulae/Casks and returns a Hash containing
+    # the version information for a resource. Returns a nil value if a latest version
+    # couldn't be found for a given resource.
+    sig {
+      params(
+        resource:  T.any(Resource),
+        json:      T::Boolean,
+        full_name: T::Boolean,
+        verbose:   T::Boolean,
+        debug:     T::Boolean,
+      ).returns(T.nilable(Hash))
+    }
+    def resource_version(
+      resource,
+      json: false,
+      full_name: false,
+      verbose: false,
+      debug: false
+    )
+      has_livecheckable = resource.livecheckable?
+
+      if debug
+        puts <<~EOS
+
+        EOS
```

Comment:
```suggestion
        puts "\n\n"
```

If you're simply looking to print a blank line, calling `puts` without an argument will accomplish this (you can see this approach used in `#latest_version`).

That said, what do we think about using a double blank line before resources to better separate the debug output? With a single blank line, resource output coming after the formula output (or another resource) tends to blend together and I feel like the extra blank line helps to keep it intelligible.

Reply:
I tend to agree with you on this one.